app/views/users/index.html.erb

```diff
@@ -1 +1,30 @@
-<h2>List of all users</h2>
\ No newline at end of file
+<h1 class="text-center mt-2">List of all users</h1>
+<% if @users.empty? %>
+<div class="mt-3 d-flex align-items-center flex-column">
+  <div class="card mb-3 border-danger" style="width: 540px">
+    <div class="card-header bg-danger text-white">No User Found</div>
+    <div class="card-body">
+      <h5 class="card-title text-danger">
+        There is no user please come again.
+      </h5>
+    </div>
+  </div>
+</div>
+<% end %>
+<div class="mt-3 d-flex align-items-center flex-column">
+  <div style="width: 540px">
+    <% @users.each do |user| %>
+    <div class="card mb-3 border-primary" style="width: 680px">
+      <div class="col-md-3">
+        <img src="<%= user.photo %>" alt="avatar" />
+      </div>
+      <div class="mt-3 d-flex align-items-center flex-column">
+        <h2 class="card-title"><%= link_to user.name , user %></h2>
+        <p class="d-flex justify-content-end text-muted" style="width: 350px">
+          Number of posts: <%= user.posts_counter %>
```

Comment:
- [ ]  kindly change `posts_counter` to `postsCounter` as it is what you used in your schema